From 67f8f900a7cc904bd06085b6c3eed5fb7f68fb38 Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 2 Jan 2022 22:34:19 +0000
Subject: [PATCH 5/6] gnu: Add asdf-wcs-schemas

* gnu/packages/astronomy.scm: (python-asdf-wcs-schemas): New variable.
---
 gnu/packages/astronomy.scm | 30 ++++++++++++++++++++++++++++++
 1 file changed, 30 insertions(+)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 0bd0c05afa..e3c7495ed4 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -1324,6 +1324,36 @@ (define-public python-asdf-coordinates-schemas
     (description "ASDF coordinates schemas")
     (license license:bsd-3)))
 
+(define-public python-asdf-wcs-schemas
+  (package
+    (name "python-asdf-wcs-schemas")
+    (version "0.1.1")
+    (source
+     (origin
+       (method url-fetch)
+       (uri (pypi-uri "asdf_wcs_schemas" version))
+       (sha256
+        (base32 "0khyab9mnf2lv755as8kwhk3lqqpd3f4291ny3b9yp3ik86fzhz1"))))
+    (build-system python-build-system)
+    (arguments
+     ;; FIXME: (Sharlatan-20211229T155217+0000): tox can't find setuptools,
+     ;; check if running tests with tox is still required.
+     (list #:tests? #f))
+    (native-inputs
+     (list python-semantic-version
+           python-setuptools-scm
+           python-tox))
+    (inputs
+     (list python-asdf))
+    (home-page "https://github.com/asdf-format/asdf-wcs-schemas")
+    (synopsis "ASDF WCS Schemas")
+    (description "World Coordinate System (WCS) as the Advanced Scientific Data
+Format (ASDF) schemas")
+    ;; FIXME: (Sharlatan-20220102T211658+0000): Check the lisense type
+    ;; Copyright (C) 2021 Association of Universities for Research in Astronomy (AURA)
+    ;; https://github.com/asdf-format/asdf-wcs-schemas/blob/main/LICENSE
+    (license license:bsd-3)))
+
 (define-public python-asdf-astropy
   (package
     (name "python-asdf-astropy")
-- 
2.34.0

